### PR TITLE
Cosmetic fixes for the SystemInfo

### PR DESCRIPTION
**Pull request by bookwar**
This patch doesn't add any new functionality. It only refactors some cosmetics around the SystemInfo object, such as

* fixes a typo in the doc string
* moves static object out of the class definition
* simplifies the logging in one function
* handles distribution_id consistently with other params.

Each change is done via a separate commit, so they can be reviewed separately or alltogether. 

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [ ] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [x] PR title explains the change from the user's point of view
- [x] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: convert2rhel/systeminfo.py [hunk 1]
```diff
@@ -60,22 +72,15 @@ def __init__(self):
...
-        #  https://access.redhat.com/security/team/key/
-        self.fingerprints_rhel = [
-            # RPM-GPG-KEY-redhat-release
-            "199e2f91fd431d51",
-            # RPM-GPG-KEY-redhat-legacy-release
-            "5326810137017186",
-            # RPM-GPG-KEY-redhat-legacy-former
-            "219180cddb42a60e",
-        ]
+        self.fingerprints_rhel = RHEL_FINGERPRINTS
```
Comment (abadger) [inline convert2rhel/systeminfo.py:83]: I would just get rid of this attribute as the module level constant is a valid replacement.

**Review by abadger — COMMENTED** [review]
One thing to change WRT RHEL_FINGERPRINTS.  The rest looks good.

File: convert2rhel/systeminfo.py (hunk 1 above)
Comment (r0x0d) [reply convert2rhel/systeminfo.py:83]: There seems to not have a lot of places calling this property, so +1 to this suggestion.

```bash
(.venv3) [r0x0d@fedora convert2rhel]$ rg fingerprints_rhel
convert2rhel/systeminfo.py
71:        self.fingerprints_rhel = [

convert2rhel/pkghandler.py
339:        system_info.fingerprints_orig_os + system_info.fingerprints_rhel
590:    non_red_hat_pkgs = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel)
641:        pkg_objects = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel, pkg)
711:        non_rhel_kernels = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel, "kernel")
823:    installed_rhel_kernels = get_installed_pkgs_by_fingerprint(system_info.fingerprints_rhel, name="kernel")
829:    non_rhel_kernels = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel, "kernel*")

convert2rhel/unit_tests/pkghandler_test.py
1831:    monkeypatch.setattr(system_info, "fingerprints_rhel", ["rhel_fingerprint"])
```

Comment (SpyTec) [reply convert2rhel/systeminfo.py:83]: Agreed! But we can do that in a follow-up PR

**Review by SpyTec — APPROVED** [review]

**Comment by SpyTec** [comment]
> One thing to change WRT RHEL_FINGERPRINTS. The rest looks good.

WDYM with WRT

**Pull request merged**